From bdd1d82af2c8f55b01fb1dafbf9f4be832ebb573 Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Fri, 27 Nov 2020 16:26:43 +0800
Subject: [PATCH] arm64: configs: rockchip_defconfig: remove
 CONFIG_PCIEASPM_POWERSAVE

ASPM is not a stable optional function for All devices from history,
and make us hard to debug any conflict issues. So remove it from default
configs.

Change-Id: I3417889dd4ab78fdc0a73d25f2e83db115558e89
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index c8186d1810fb..01cfa76456ef 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -47,7 +47,6 @@ CONFIG_PROFILING=y
 CONFIG_ARCH_ROCKCHIP=y
 CONFIG_PCI=y
 CONFIG_PCIEPORTBUS=y
-CONFIG_PCIEASPM_POWERSAVE=y
 CONFIG_PCIE_ROCKCHIP_HOST=y
 CONFIG_PCIE_DW_ROCKCHIP=y
 # CONFIG_ARM64_ERRATUM_826319 is not set
-- 
2.35.3

